src/main/kotlin/org/opensearch/observability/ObservabilityPlugin.kt

```diff
@@ -81,9 +85,17 @@ class ObservabilityPlugin : Plugin(), ActionPlugin, JobSchedulerExtension {
     ): Collection<Any> {
         PluginSettings.addSettingsUpdateConsumer(clusterService)
         ObservabilityIndex.initialize(client, clusterService)
+        ObservabilityMetricsIndex.initialize(client, clusterService)
+        ObservabilityTracesIndex.initialize(client, clusterService)
```

Comment:
here

Reply:
Only when the Observability migration would start we will the usage of new .integration index
I dont want to mix too many things into this PR - its already large enough

Reply:
Integration would be marked as a mandatory plugin